### PR TITLE
feat: add contributing guidelines

### PR DESCRIPTION
**Pull request by manel1874**
Added contributing guidelines as a separate file.

### CODE REVIEW

File: CONTRIBUTING
Comment (jcabrero) [inline CONTRIBUTING:1]: Shouldn't this file be named `CONTRIBUTING.md`?

File: README.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ poetry install
 poetry run pytest
 ```
 
+## Join Nillion community and contribute
+
+Please join the Nillion community on [Discord](https://discord.com/invite/nillionnetwork).
+
+See the [CONTRIBUTING](./CONTRIBUTING) file for how to contribute to `nada-numpy`.
```
Comment (mathias-nillion) [inline README.md:44]: @manel1874 this link is now dead

**Review by jcabrero — APPROVED** [review]
Thanks a lot! This looks very nice.

**Review by mathias-nillion — APPROVED** [review]
LGTM - just one dead link in doc (small typo)

**Pull request merged**